### DISCUSSION TITLE
run make error with `ERRO Running error: context loading failed: no go files to analyze`

### DISCUSSION THREAD

**Discussion by JooKS-me** [kubevela/kubevela#3185] [Q&A] [answered]
Hi community,
Hope you can help me with this problem. QAQ... There are some problems when I refer to https://github.com/oam-dev/kubevela/blob/master/contribute/developer-guide.md to build Vela CLI.

After I type `make` into the terminal, the following information is displayed:
````
go fmt ./...
/Users/zhukunshuai/go-project/bin/goimports -local github.com/oam-dev/kubevela -w $(go list -f {{.Dir}} ./...)
/opt/homebrew/bin/cue fmt ./vela-templates/definitions/internal/*
/opt/homebrew/bin/cue fmt ./vela-templates/definitions/deprecated/*
/opt/homebrew/bin/cue fmt ./vela-templates/definitions/registry/*
/opt/homebrew/bin/cue fmt ./pkg/stdlib/pkgs/*
/opt/homebrew/bin/cue fmt ./pkg/stdlib/op.cue
/opt/homebrew/bin/cue fmt ./pkg/workflow/tasks/template/static/*
go vet ./...
22:56:23 [ OK ] golangci-lint is already installed
/Users/zhukunshuai/go-project/bin/golangci-lint run ./…
ERRO Running error: context loading failed: no go files to analyze
make: *** [lint] Error 5
````

#### My environment
My computer is apple with arm architecture.
go version is `go version go1.16.13 darwin/arm64`.

Also, found this according to my search for golangci's issue: https://github.com/golangci/golangci-lint/issues/825.
However, whether GO111MODULE is set to 'on' or 'off', the above error is displayed.

**Comment by JooKS-me** [comment, marked as answer]
It works fine when I execute the command:
```shell
golangci-lint cache clean && sudo go clean -modcache -cache -i
```

**Reply by wonderflow** [reply]
Cool, thanks for sharing
